css/admin-theme.css

```diff
@@ -0,0 +1,196 @@
+@import url("https://fonts.googleapis.com/css2?family=DM+Mono:wght@300;400;500&display=swap");
+
+:root {
+  --peacock-green: #376c74;
+  --mediterranean-blue: #187e9b;
+  --robin-egg: #d9f1f3;
+  --wool: #fbfbf6;
+  --light-gray: #d4d4d4;
+  --darkest-gray: #1f1f1f;
+  --gin-font: "DM Mono", monospace;
+  --gin-font-size-h1: 1.75rem;
+  --gin-font-size-h2: 1.5rem;
+  --gin-font-size-h3: 1.25rem;
+  --gin-border-m: 0.313rem;
+}
+
+:root [data-gin-accent=light_blue] {
+  --gin-color-primary-rgb: 55, 108, 116;
+  --gin-color-primary-active: var(--peacock-green);
+  --gin-color-secondary: var(--mediterranean-blue);
+  --gin-color-primary-hover: var(--peacock-green);
+  --gin-color-primary-light: var(--robin-egg);
+  --gin-border-color-layer: var(--light-gray);
+  --gin-bg-layer: var(--wool);
+  --gin-dark-gray: var(--darkest-gray);
+  --colorGinText: var(--darkest-gray);
+  --gin-border-color-table-header: var(--darkest-gray);
+}
+
+.gin--dark-mode [data-gin-accent=light_blue] {
+  --gin-color-primary-rgb: 55, 108, 116 !important;
+  --gin-color-primary-active: var(--peacock-green) !important;
+  --gin-color-secondary: var(--mediterranean-blue) !important;
+  --gin-color-primary-hover: var(--peacock-green) !important;
+  --gin-color-primary-light: var(--robin-egg) !important;
+  --gin-border-color-layer: var(--light-gray) !important;
+  --gin-bg-layer: var(--darkest-gray) !important;
+  --gin-border-color-table-header: var(--darkest-gray) !important;
+  --gin-bg-input: #fff !important;
+  --gin-border-color-form-element: var(--light-gray) !important;
+  --gin-bg-app: #353535;
+  --gin-color-text-light: var(--light-gray);
+  --gin-color-primary-light-active: var(--robin-egg);
+  --gin-icon-color: var(--light-gray);
+  --gin-bg-layer2: #fff;
+  --gin-bg-layer3: #fff;
+  --gin-color-button-text: #fff;
+}
+
+:root h1,
```

Comment:
I understand the need to include the admin-css on the front-end, however, I think this h1 is leaking through and causing the page title to become more bolded than it used to be.
![Screenshot 2023-05-03 at 9 27 39 AM](https://user-images.githubusercontent.com/107938318/235979771-08e33716-13d6-435b-a8fa-a827b89da6db.png)